contributor-guide/modules/ROOT/pages/testing/continuous-integration.adoc

```diff
@@ -1,71 +1,94 @@
 = Continuous Integration
 
+
 The Boost project uses Continuous Integration (CI) to ensure the quality and integrity of its code. CI is the practice of merging all developers' working copies to a shared mainline several times a day. The main aim is to prevent integration problems, which are detected and can be fixed as early as possible.
 
 Boost uses several CI services for testing on different platforms and compilers. Many libraries use two or three of the systems described here, as does the https://github.com/boostorg/boost/tree/master[Super-project] itself.
 
 Note:: It is a requirement for a new library submission to Boost to include an appropriate CI system. Refer to the examples for each CI system to better understand what is involved.
```

Comment:
```suggestion
Note:: It is a requirement for a new library submission to Boost to include an appropriate CI/CD system. Refer to the examples for each CI/CD system to better understand what is involved.
```

Reply:
I am leaving it as CI for now - not sure we use the CD side of these tools as the releases are not updated? Open to correction.